### PR TITLE
Isogram approaches

### PR DESCRIPTION
**Pull request by bobahop**
Initial commit. Will need to run format and tweak a bit more.

### CODE REVIEW

**Comment by bobahop** [comment]
Please

/format

me

**Comment by bobahop** [comment]
@ErikSchierboom JS Isogram approaches is ready for review

File: exercises/practice/isogram/.approaches/introduction.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## General guidance
+
+The key to solving Isogram is to determine if any of the letters in the `string` being checked are repeated.
+A repeated letter means the `string` is not an Isogram.
+The occurrence of the letter `a` and the letter `A` count as a repeated letter, so `Alpha` would not be an isogram.
+
+## Approach: regular expression to match a duplicated letter
+
+```javaascript
```
Comment (ErikSchierboom) [inline exercises/practice/isogram/.approaches/introduction.md:15]:
```suggestion
```javascript
```

File: exercises/practice/isogram/.approaches/regex-match-dupe/content.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Regular expression to match a duplicate letter
+
+```javaascript
```
Comment (ErikSchierboom) [inline exercises/practice/isogram/.approaches/regex-match-dupe/content.md:3]:
```suggestion
```javascript
```

**Review by ErikSchierboom — APPROVED** [review]
Two tiny nits

**Comment by bobahop** [comment]
@ErikSchierboom  Nits fixed! Thanks for catching them. I can't seem to type a complete sentence without at least one typo. I'm constantly going back over and fixing typos. It's one reason why it takes me so long to do these things. For instance, I can't tell you how many times I've typed "closure" as "closrue".

**Pull request merged**